src/testcafe-runner.js

```diff
@@ -97,6 +97,11 @@ async function runTestCafe ({projectPath, assetsPath, suite, metrics, timeoutSec
       });
     }
 
+    if (process.env.HTTP_PROXY !== undefined) {
```

Comment:
In our scenario, isn't this is equivalent to:
```suggestion
    if (process.env.HTTP_PROXY) {
```

?

Just pondering for a slightly easier readability. Approved otherwise 👍

Reply:
updating!